modules/deployment-plugin-cluster.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * web_console/dynamic-plugin.adoc
+
+:_content-type: PROCEDURE
+[id="build-image-with-docker"]
+= Build an image with Docker
+In order to deploy your plug-in on a cluster, you need to build an image and push it to an image directory.
+
+.Procedure
+
+. Build the image with the following command:
++
+[source,terminal]
+----
+$ docker build -t quay.io/my-repositroy/my-plugin:latest
+----
+
+. Run the image with the following command:
```

Comment:
This isn't strictly needed to publish the image, although it might be something the developer wants to do to test before pushing.

Reply:
I will update as an optional step.